sympy/concrete/summations.py

```diff
@@ -10,7 +10,7 @@
 from sympy.core.containers import Tuple
 from sympy.core.function import Derivative, expand
 from sympy.core.mul import Mul
-from sympy.core.numbers import Float
+from sympy.core.numbers import Float, _illegal as illegal
```

Comment:
Not sure if it was like this in master -- did you want to expose this as non-underscored? If not, there is only one line below that needs to be changed (where `n_illegal` is defined).

Reply:
This was previously imported as `illegal` (as seen lower in the diff).